hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/KeyValueStreamDataChannel.java

```diff
@@ -167,11 +163,9 @@ public int write(ReferenceCountedObject<ByteBuffer> referenceCounted)
     getMetrics().incContainerOpsMetrics(getType());
     assertOpen();
 
-    final long l = Time.monotonicNow();
-    int len = writeBuffers(referenceCounted, buffers, super::writeFileChannel);
-    getMetrics()
-        .incContainerOpsLatencies(getType(), Time.monotonicNow() - l);
-    return len;
+    final long start = Time.monotonicNow();
```

Comment:
`writeBuffers` runs in a loop but using the same `start` time.  The time will be double count when there are more than one buffers.